app/bundles/CoreBundle/Assets/js/1.core.js

```diff
@@ -67,6 +67,30 @@ mQuery(document).ready(function() {
         });
     }
 
+    // Absurdly aggressive method to ensure tooltips always work anywhere
```

Comment:
A bit of a similar issue as I described in #13948 : the code looks fine, but it shouldn't be in this location.
I'm fine with tooltips code existing in `core.js` if there's no better place for it, but you need to follow the style and formatting of the rest of the file:

- functions should be in the `Mautic` namespace (nesting starts around line 154, or you can prefix it like happens in other files)
- variables should be prefixed with `MauticVars` (as you can see above that)

Reply:
Oh, got it
It’ll be fixed asap

I’ll also take notes for future implementations 
Thanks for reviewing 🙏🏻